### PR TITLE
chore: upgrade package

### PR DESCRIPTION
**Pull request by wzh425**

### CODE REVIEW

File: src/Services/Masa.Mc.Service/Program.cs [hunk 1]
```diff
@@ -120,7 +116,11 @@
             dbOptions => dbOptions.UseSqlServer().UseFilter())
         .UseRepository<McDbContext>();
     })
-    .AddServices(builder);
+    .AddServices(builder, options =>
+    {
+        options.GetPrefixes = new() { "Get", "Select", "Find" };
```
Comment (doddgu) [inline src/Services/Masa.Mc.Service/Program.cs:121]: I don't think need this.

File: src/Web/Masa.Mc.Web.Admin.Server/Program.cs [hunk 2]
```diff
@@ -19,17 +19,24 @@
...
+#endif
 
-builder.Services.AddAuthApiGateways(option => option.McServiceBaseAddress = builder.Configuration["McServiceBaseAddress"]);
-builder.Services.AddMasaStackComponentsForServer("wwwroot/i18n", builder.Configuration["AuthServiceBaseAddress"], builder.Configuration["McServiceBaseAddress"]);
 builder.Services.AddHttpContextAccessor();
 builder.Services.AddGlobalForServer();
-builder.Services.AddElasticsearchClient("auth", option => option.UseNodes("http://10.10.90.44:31920/").UseDefault())
-                .AddAutoComplete(option => option.UseIndexName(builder.Configuration["UserAutoComplete"]));
+builder.Services.AddScoped<TokenProvider>();
+//builder.Services.AddElasticsearchClient("auth", option => option.UseNodes("http://10.10.90.44:31920/").UseDefault())
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin.Server/Program.cs:34]: Why comment it?

File: src/Web/Masa.Mc.Web.Admin/App.razor [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
-            <Masa.Stack.Components.Others.NotFound/>
-        </MApp>
-    </NotFound>
-</Router>
+<CascadingAuthenticationState>
+    <Microsoft.AspNetCore.Components.Routing.Router AppAssembly="@typeof(App).Assembly"
+        AdditionalAssemblies='new List<Assembly>(){ Assembly.Load("Masa.Stack.Components")}'>
+        <Found Context="routeData">
+            <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
+            <FocusOnNavigate RouteData="@routeData" Selector="h1" />
```
Comment (doddgu) [inline src/Web/Masa.Mc.Web.Admin/App.razor:8]: I think it don't need h1 selector.

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Mc.Service/Services/MessageTaskService.cs [hunk 4]
```diff
@@ -5,26 +5,12 @@ namespace Masa.Mc.Service.Admin.Services;
...
-        MapPost(ResolveReceiversCountAsync);
-        MapPost(ExecuteAsync, "execute/{messageTaskId}");
-        MapPost(SendOrdinaryMessageAsync);
-        MapPost(SendTemplateMessageAsync);
+    public MessageTaskService(IServiceCollection services) : base("api/message-task")
+    {
+
     }
 
+    [RoutePattern("", StartWithBaseUri = true, HttpMethod = "Get")]
```
[the remark this answers is not in the document]
Comment (wzh425) [reply src/Services/Masa.Mc.Service/Services/MessageTaskService.cs]: This is an outdated document

File: src/Web/Masa.Mc.Web.Admin.Server/Program.cs (hunk 2 above)
Comment (wzh425) [reply src/Web/Masa.Mc.Web.Admin.Server/Program.cs]: It has been included in AddMasaStackComponentsForServer

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**